src/pages/Ingame/index.js

```diff
@@ -52,12 +52,22 @@ export default function Game() {
     setRoundDone(false);
     setIsSubmitted(false);
     emitJoinTeam(roomName);
+    socket.on("cur_result", (curResult) => {
```

Comment:
아래처럼 명시적으로 보여주는게 좋은 거 같습니다.  아래도 동일
```suggestion
    socket.on("cur_result", (currenTResult) => {
```